SayTheirNames/Source/Controller/Home/Person/PersonController.swift

```diff
@@ -9,14 +9,36 @@
 import UIKit
 
 class PersonController: BaseViewController {
-
+    @IBOutlet weak var readMoreButton: UIButton!
+    @IBOutlet weak var donateButton: UIButton!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         view.accessibilityIdentifier = "personView"
 
         // Do any additional setup after loading the view.
+        setupLocalizedStrings()
     }
     
+    private func setupLocalizedStrings() {
+        if let readMore = NSLocalizedString("Read_More", comment: "Read More") as String? {
+            self.readMoreButton.setTitle(readMore, for: UIControl.State.normal)
+            self.readMoreButton.setTitle(readMore, for: UIControl.State.selected)
+        } else {
+            self.readMoreButton.setTitle("READ MORE ->", for: UIControl.State.normal)
+            self.readMoreButton.setTitle("READ MORE ->", for: UIControl.State.selected)
+        }
+        
+        if let donate = NSLocalizedString("Donate", comment: "Donate") as String? {
+            self.donateButton.setTitle(donate, for: UIControl.State.normal)
+            self.donateButton.setTitle(donate, for: UIControl.State.selected)
+        } else {
+            self.donateButton.setTitle("Donate", for: UIControl.State.normal)
+            self.donateButton.setTitle("Donate", for: UIControl.State.selected)
+        }
```

Comment:
@PedroTrujilloV Is there a better way to include Localized strings without having `NSLocalizedString` to keep things neat??

Reply:
I can add it as an array `[.normal, .selected]` because currently the string is the same, to avoid some duplication. Is it what you mean?

Reply:
I sort of thought we were using https://github.com/SwiftGen/SwiftGen for localizations. If so these should be something like `L10n.donate`, right? I'm not sure if it's documented anywhere, but I do see [this](https://github.com/Say-Their-Name/say-their-names-ios/blob/feature/home_diffable_data_source/SayTheirNames/Resources/Generated/strings.swift) generated file of localizations.

Reply:
@rodericj Yes, and thanks for the reference 💯

Reply:
good idea to document it 💯